blkn/src/Button/IconWrapper.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import * as React from "react";
+import { colors } from "../constants";
+
+type Props = {
+  size: "normal" | "small" | "large",
+  type: "primary" | "secondary",
+  icon: Function,
+};
+
+const sizes = {
+  normal: "20px",
+  small: "20px",
+  large: "26px",
+};
+
+const fillColors = {
+  primary: colors.white,
+  secondary: colors.shuttle,
+};
+
+const IconWrapper = (props: Props) => {
+  const { icon: Icon, type, size } = props;
```

Comment:
if `Icon` is a class and not an element, make the prop itself PascalCase

Reply:
It does not work when prop is capitalized.